packages/agent/conf/examples/presentation_definitions/banqup.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
this needs to be split up. rename banqup.json to banqup-account.json . revert the content to the original. and add a banqup-identity.json

Reply:
and for now keep english as default i would say